site-admin/templates/admin/save.hbs

```diff
@@ -0,0 +1,122 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Admin | KtmJS</title>
+    <link rel="stylesheet" href="/admin/css/admin.css" type="text/css"/>
+  </head>
+
+  <body>
+    <div class="Panel__Navigation">
+      <img class="logo" src="/admin/images/logo.png" alt="Kathmandu JS User Group Logo" />
+      <ul class="Navigation">
+        <li><a href="/meetups">Meetups</a></li>
+        <li><a href="/companies">Companies</a></li>
+        <li><a href="/people">People</a></li>
+      </ul>
+    </div>
+
+    <div class="Panel__Timeline">
+      <form action="/meetups" method="post">
+        <input type="text" placeholder="episode" name="episode" value={{episode}} />
+        <input type="date" placeholder="date" name="date" />
+        <h5>Starting time</h5>
+        <input type="time" placeholder="starting time" name="startsAt" />
+        <h5>Ending time</h5>
+        <input type="time" placeholder="ending time" name="endsAt" />
+
+        <h4>Venue</h4>
+        <input type="text" placeholder="Organisation" name="venue[organisation]" />
+        <input type="text" placeholder="Address Line 1" name="venue[addr_line1]" />
+        <input type="text" placeholder="Address Line 2" name="venue[addr_line2]" />
+
+        <h4>Sessions</h4>
+        <div id="sessions">
+          <input type="text" placeholder="title">
+          <input type="time" placeholder="time">
+          <input type="text" placeholder="desc">
+          {{{dropdown "people"}}}
+          <button onclick="return addPeople(event);">Add People</button>
+        </div>
+        <button onclick="return addSession(event);">Add Session</button>
+
+        <h4>Sponsors</h4>
+        <div id="sponsors">
+          {{{dropdown "companies"}}}
+        </div>
+        <button onclick="return addSponsors(event);">Add Sponsors</button>
+
+        <h4>Supporters</h4>
+        <div id="supporters">
+          {{{dropdown "companies"}}}
+        </div>
+        <button onclick="return addSupporters(event);">Add Supporters</button>
+
+        <input type="submit" value="Save" />
+      </form>
+
+      <script>
```

Comment:
Move the scripts to a file. ignore if you planning to do a refactor before merge.